packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -127,20 +129,7 @@ contract PluginRepo is
         bytes calldata _buildMetadata,
         bytes calldata _releaseMetadata
     ) external auth(address(this), MAINTAINER_PERMISSION_ID) {
-        // In a case where _pluginSetup doesn't contain supportsInterface,
-        // but contains fallback, that doesn't return anything(most cases)
-        // the below approach aims to still return custom error which not possible with try/catch..
-        // NOTE: also checks if _pluginSetup is a contract and reverts if not.
```

Comment:
Rationale for removing this? Same effect without it?

Reply:
The new way now is even  safer. I had this to write better and wanted to do the last step, but I found out OZ already fixed it so here it is.